in-depth/submission_guidelines.md

```diff
@@ -0,0 +1,177 @@
+* Theming Extensions (`theming`) - A plugin that adds extends twig by adding extra variables or functionality.
+
+### Meta
+
+A short string of descriptive text that displays below your heading when you mouse over your item.
+
+### Link
+
+The URL of the "download" link, at the bottom of your description.
+
+### Images
+
+These are your `fullsize` images.  Specify the absolute location of each image.
+
+### Info
+
+Here you can specify a few lines of "info" that will display above your description.  The recommended items are `By` and `Last Updated`, but you can create your own items if they'd better fit the context of your theme or plugin.
+
+If your entry is a port of someone else's work, please use the `Ported By` and `Original By` lines instead of the regular `By`.  We'd like to make sure that the original content creators receive proper credit for their work.
+
+Also, as visible in the template below, you can add links to these `info` items.  You can easily link your `By` line to your GitHub profile or another location.  If you're using the `Original By` line, you should also link to the original source in this manner.  You can use either HTML or Markdown links, but if you use Markdown, you'll want to wrap the entire string in quotes.  This is because the `[]` characters have a special meaning in YAML and must be escaped.
+
+You can also use `_blank:` to create a line that doesn't start with an identifier.
+
+### Description
+
+Finally, your description should go after the YAML, in the Markdown body.  Your description can be as long as you'd like, but keep in mind that the description area is very narrow (it's roughly 40 characters wide).
+
+## Markdown/YAML Template
+
+The following code block is a template for your submission's `.md` file.  Copy and paste this section into your document and edit it to create your submission.
+
+```
+---
+heading: My Theme or Plugin
+thumbnail: /_themes/images/thumbnails/my_theme_or_plugin.jpg
+categories:
+  - category1
+  - category2
+meta: A short description for the hover text.
+link: https://github.com/octocat/Spoon-Knife
+images:
+  - /_themes/images/thumbnails/my_theme_or_plugin.jpg
+#  - /_themes/images/thumbnails/my_theme_or_plugin-2.jpg
+#  - /_themes/images/thumbnails/my_theme_or_plugin-3.jpg
+info:
+  By: "[The Octocat](https://github.com/octocat)"
+#  Ported By: "[The Octocat](https://github.com/octocat)"
+#  Original By: "[Someone Else](http://example.com/)"
+  Last Updated: They don't have to be "By" and "Last Updated"
+---
+This is my Theme for Pico.
+
+Or maybe it's a Plugin...
+
+Either way, if it really existed, this is where I would write some details about it.
+```
+
+## Getting Started
+
+* This section will contain links to:
+* Info on how to fork and edit on github
+* Info on making a pull Request
+* Probably shy away from any actual `git` content.  Presumably, knowing how to use GitHub will suffice for this process.
```

Comment:
Maybe, I don't remember, but as you've said, I actually didn't read it before. However, I think that such a help page is a absolutely great idea - i.e. to help people so that they can help us :laughing: